tgui/packages/tgui/interfaces/ComponentPrinter.tsx

```diff
@@ -2,7 +2,7 @@ import { createSearch } from 'common/string';
 import { useBackend, useLocalState } from '../backend';
 import { Material, MaterialAmount, MaterialFormatting, Materials, MATERIAL_KEYS } from './common/Materials';
 import { Window } from '../layouts';
-import { Button, Input, Section, Stack, Tabs } from '../components';
+import { Button, Input, Section, Stack, Tabs, Box } from '../components';
```

Comment:
```suggestion
import { Box, Button, Input, Section, Stack, Tabs } from '../components';
```

my beautiful abc order